include/wx/string.h

```diff
@@ -2132,6 +2132,11 @@ class WXDLLIMPEXP_BASE wxString
       // insert a double into string
   wxString& operator<<(double d)
     { return (*this) << Format(wxT("%g"), d); }
+  // insert a pointer as hex into string
+  wxString &operator<<(void *p)
+  {
+      return (*this) << Format(wxT("%p"), p);
```

Comment:
```suggestion
      return (*this) << Format(wxS("%p"), p);
```
wxT() should not be used in the new code.

Reply:
Updated with  fe4b205

Reply:
Shall I replace the other `wxT` occurences in that file as well?

Reply:
> Shall I replace the other `wxT` occurences in that file as well?

I do not think it is usually done, all the irrelevant changes would make the PR harder to review.